openmc/deplete/cram.py

```diff
@@ -75,9 +75,9 @@ def __call__(self, A, n0, dt):
             Final compositions after ``dt``
 
         """
-        A = sp.csr_matrix(A * dt, dtype=np.float64)
+        A = dt * sp.csc_matrix(A, dtype=np.float64)
         y = n0.copy()
-        ident = sp.eye(A.shape[0])
+        ident = sp.eye(A.shape[0], format='csc')
```

Comment:
From your comments, it seems this is the main source of the speedup. Having consistent array layouts likely helps the `A - theta * ident` step later.

But would we get similar speed up from forcing `format="csr"`?

Reply:
Yes we would get a similar speed up. The reason I'm trying to ultimately get to the `splu` objects is so we can cache them if the source rate and timestep don't change which would give a substantial speed up for `IndependentOperator` calculations with multiple decay steps.

Reply:
> The reason I'm trying to ultimately get to the CSC format and splu objects is so we can cache them

Oh this is exciting news! Yeah if your depletion matrix doesn't change from time to time, storing the factorized forms makes a ton of sense.

One thing we could change prior to this would be using a csr format for the identity matrix. If that doesn't change regression outputs, that yields a benefit to users sooner while giving us time to debug the csc + ci related changes.